src/bot/precomileInclude.jl

```diff
@@ -0,0 +1,114 @@
+
+"""
+    precompilePather(packageName::String)
+
+To get the path of precompile_packageName.jl file
+
+Written exclusively for SnoopCompile Github actions.
+# Examples
+```julia
+precompilePath, precompileFolder = precompilePather("MatLang")
+```
+"""
+function precompilePather(packageName::String)
+    return "\"../deps/SnoopCompile/precompile/precompile_$packageName.jl\"",
```

Comment:
Should this be something like `joinpath(@__DIR__, "..", "deps", ...)` to be independent of the current working directory?